cpp/include/cuml/svm/svc.hpp

```diff
@@ -20,6 +20,7 @@
 #include "svm_parameter.h"
 #include <cuml/common/logger.hpp>
 #include <raft/core/handle.hpp>
+#include <raft/distance/detail/matrix/matrix.hpp>
```

Comment:
I'm not sure why we're doing this when we have mdspan and the new sparse formats available. Do we `really` need a single matrix format just to unify these? Why not just use overloads for the sparse and dense formats we have?

Reply:
We also shouldn't be invoking any APIs in `detail` outside of RAFT because these are not stable and will change without warning. There are a couple places where we've had to do this in cuML only while we migrate these APIs in RAFT but will be cleaning those up at some point soon.